lib/internal/errors.js

```diff
@@ -75,6 +75,8 @@ const kTypes = [
 const MainContextError = Error;
 const overrideStackTrace = new SafeWeakMap();
 const kNoOverride = Symbol('kNoOverride');
+let userStackTraceLimit = Error.stackTraceLimit;
```

Comment:
This assignment is not useful, is it?
```suggestion
let userStackTraceLimit;
```

Reply:
You are right, it is a leftover from the initial version in which a user exception that came before the first Node exception could choke on the `undefined` value - it is impossible with the current code - but it doesn't cost anything and it could transform into a trap in the future

Reply:
It doesn't cost anything, but if you don't assign the value here, it makes it clearer that the code doesn't care of the value of `Error.stackTraceLimit` at this point. Also, if the code is changed in the future and introduced back the old behavior, I think it's helpful if it crashes to spot the mistake.